inference-engine/ie_bridges/java/tests/src/Test.java

```diff
@@ -0,0 +1,24 @@
+package src;
+
+import junit.framework.TestCase;
+
+import org.intel.openvino.*;
+
+public class Test extends TestCase {
+    ConfigTests config = new ConfigTests();
+    String exceptionMessage;
+
+    Test(){
+        try {
+            System.loadLibrary(IECore.NATIVE_LIBRARY_NAME);
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Failed to load Inference Engine library\n" + e);
+            System.exit(1);
+        }
```

Comment:
Is that OK to load the library here but not in `setUp`? I mean does it work correctly?

Reply:
I think it's more correct than load library in ```setUp``` method, because ```setUp``` is called before the invocation of each test. Yes, now it's work correctly.